portfolio/src/main/webapp/map.html

```diff
@@ -13,8 +13,8 @@
     <body onload="initMap();">
         <nav>
     		<ul class="sticky">
```

Comment:
Same here, the indentation needs to be updated.

Reply:
The indentation is updated.